x/cronos/types/interfaces.go

```diff
@@ -76,3 +79,10 @@ type EvmKeeper interface {
 	) (sdk.Coins, error)
 	ChainID() *big.Int
 }
+
+// ICS4Wrapper defines the expected ICS4Wrapper for middleware
+type ICS4Wrapper interface {
```

Comment:
this seems not used anywhere?

Reply:
yes this can be removed for now (this is needed if we need to write a "full" middleware)